iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/TopologyModificationUtils.java

```diff
@@ -230,6 +239,18 @@ static void createNodeBreakerSwitches(int node1, int middleNode, int node2, Stri
         createNBDisconnector(middleNode, node2, suffix, prefix, view, open);
     }
 
+    static void createNBBreaker(int node1, int node2, String id, VoltageLevel.NodeBreakerView view, boolean open) {
+        view.newSwitch()
+                .setId(id)
+                .setEnsureIdUnicity(true)
+                .setKind(SwitchKind.BREAKER)
+                .setOpen(open)
+                .setRetained(true)
+                .setNode1(node1)
+                .setNode2(node2)
+                .add();
+    }
+
     static void createNBBreaker(int node1, int node2, String suffix, String prefix, VoltageLevel.NodeBreakerView view, boolean open) {
```

Comment:
To avoid code duplication, this method should call your new `createNBBreaker` method.